linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_emergency_expire_date/rule.yml

```diff
@@ -0,0 +1,52 @@
+documentation_complete: true
+
+prodtype: fedora,rhel8
+
+title: 'Assign Expiration Date to Emergency Accounts'
+
+description: |-
+    Emergency accounts are privileged accounts established in response to
+    crisis situations where the need for rapid account activation is required.
+    In the event emergency accounts are required, configure the system to
+    terminate them after a documented time period. For every emergency account,
+    run the following command to set an expiration date on it, substituting
+    <tt><i>ACCOUNT_NAME</i></tt> and <tt><i>YYYY-MM-DD</i></tt>
+    appropriately:
+    <pre>$ sudo chage -E <i>YYYY-MM-DD ACCOUNT_NAME</i></pre>
+    <tt><i>YYYY-MM-DD</i></tt> indicates the documented expiration date for the
+    account. For U.S. Government systems, the operating system must be
+    configured to automatically terminate these types of accounts after a
+    period of 72 hours.
+
+rationale: |-
+    If emergency user accounts remain active when no longer needed or for
+    an excessive period, these accounts may be used to gain unauthorized access.
+    To mitigate this risk, automated termination of all emergency accounts
+    must be set upon account creation.
+    <br />
+
+severity: medium
+
+identifiers:
+    cce@rhel8: CCE-85910-8
+
+references:
+    cis-csc: 1,12,13,14,15,16,18,3,5,7,8
+    cobit5: DSS01.03,DSS03.05,DSS05.04,DSS05.05,DSS05.07,DSS06.03
+    disa: CCI-000016,CCI-001682
+    isa-62443-2009: 4.3.3.2.2,4.3.3.5.1,4.3.3.5.2,4.3.3.7.2,4.3.3.7.3,4.3.3.7.4
+    isa-62443-2013: 'SR 1.1,SR 1.2,SR 1.3,SR 1.4,SR 1.5,SR 1.7,SR 1.8,SR 1.9,SR 2.1,SR 6.2'
+    iso27001-2013: A.12.4.1,A.12.4.3,A.6.1.2,A.7.1.1,A.9.1.2,A.9.2.1,A.9.2.2,A.9.2.3,A.9.2.4,A.9.2.6,A.9.3.1,A.9.4.1,A.9.4.2,A.9.4.3,A.9.4.4,A.9.4.5
+    nist: AC-2(2),AC-2(3),CM-6(a)
```

Comment:
Checking on https://commoncriteria.github.io/pp/references/nistvscci.html it only lists AC-2 (2) for the two CCIs listed. But I see the case for leaving them there.

Reply:
This was literally just a copy/paste from an existing rule, so I'm assuming those references were "correct".. the only difference is this rule is more explicit of using the term "emergency" account, rather than temporary.